tests/phpunit/tests/block-bindings/render.php

```diff
@@ -157,6 +157,94 @@ public function test_passing_uses_context_to_source() {
 		);
 	}
 
+	/**
+	 * Tests passing `uses_context` as argument to the source.
+	 *
+	 * @ticket 99999
```

Comment:
This needs to change before committing.

Reply:
Added in this commit: https://github.com/WordPress/wordpress-develop/pull/6456/commits/986ff291501e3598efb32e93b50fef42f798eebd